src/services/BuildRequester.groovy

```diff
@@ -9,53 +9,22 @@ import main.exception.TravisHelperException
 import main.util.FileManager
 import static main.app.MiningFramework.arguments
 
-class BuildRequester implements DataCollector {
+abstract class BuildRequester implements DataCollector {
```

Comment:
isso vai ser integrado no framework? se for essa mudança pode quebrar outras implementações